Watts-Lab/commonsense-platform
# 145 language support for experiment website

Description:
This pull request adds language support for 10 different languages, allowing the text on the main experiment website to be translated into a language selected by the user. 

It also integrates the locize service for localization management. This integration will enable automatic checks and updates for translations, as well as validation checks for new translations, streamlining the localization process.

How to test features:
1. Change the language in the LocaleSwitcher component and verify that all text on the main experiment website is translated correctly.
2. Ensure that the automatic translation updates and validation checks are functioning as expected with Locize.

Feel free to set up your own account/project with locize! More about locize can be found in their documentation: https://docs.locize.com/
